op-challenger/config/config_test.go

```diff
@@ -24,6 +24,14 @@ var (
 	validRollupRpc             = "http://localhost:8555"
 )
 
+var (
+	validAsteriscBin             = "./bin/asterisc"
```

Comment:
nit: merge these blocks. Can delineate with an empty line between for legibility.

```suggestion

	validAsteriscBin             = "./bin/asterisc"
```

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/10094/commits/ceb1b24643cb3cc9c50f7fc1c9b90f15096a729c.